README.md

```diff
@@ -37,3 +37,37 @@ If you want to run production build:
 > npm run build
 > npm run preview
 ```
+
+---------------------
+## Backend
+### With docker
+Production build can be run by Docker. To build image go to `/backend/Car-Detailing-Backend` and write:
+```bash
+> docker build -t car-detailing-backend .
+```
+Then run image
+```bash
+> docker run -p 32778:443 car-detailing-backend
+```
```

Comment:
Unfortunately, it only runs on the `80` port instead of the `443` when Docker runs it :crying_cat_face: 
So please adjust the description